addons/stock/wizard/stock_warn_insufficient_qty_views.xml

```diff
@@ -6,7 +6,7 @@
         <field name="arch" type="xml">
             <form>
                 <div>
-                    The product is not available in sufficient quantity
+                    The product is not available in sufficient quantity.
```

Comment:
? There more to the sentence after "quantity" when multi-locations are active, so there is purposely no .
![image](https://github.com/odoo/odoo/assets/57942143/354e8e86-123d-43c6-a05a-ef87b4c49287)